src/components/Flex/Flex.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useRef, forwardRef } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import useMergeRefs from "../../hooks/useMergeRefs";
+import { FLEX_POSITIONS, FLEX_SPACING_SIZES } from "./FlexConstants";
+import classes from "./Flex.module.scss";
+import { BASE_POSITIONS } from "../../constants/positions";
+
+const Flex = forwardRef(
+  (
+    {
+      className,
+      id,
+      vertical,
+      wrap,
+      children,
+      horizontalPosition,
+      verticalPosition,
+      horizontalSpacingSize,
+      verticalSpacingSize,
+      style
+    },
+    ref
+  ) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+
+    return (
+      <div
```

Comment:
lets add the ability to pass whatever element that is needed but give it a default of `div`